requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit==0.13.3
```

Comment:
Maybe adding something like the following in the README will actually add more visibility into Yelp's pre-commit project and provide a better understanding to the developer, rather than using a `requirements.txt` file that might confuse collaborators :)

* * *

# Collaborate

This project is using [pre-commit](http://pre-commit.com/) to help the developer run pre-commit hooks easily, allowing to [autoformat Python files](https://pypi.python.org/pypi/autopep8), [reorder the imports](https://pypi.python.org/pypi/isort), and other various checks.

You can easily start using it via `pip install pre-commit`.